tests/dummy/config/environment.js

```diff
@@ -49,7 +49,7 @@ module.exports = function(environment) {
   }
 
   if (environment === 'production') {
-    ENV.rootURL = '/pages/open-source-staging/ember-nrg-ui';
+    ENV.rootURL = '/ember-nrg-ui';
```

Comment:
Is this change to convert from an internal relative path to a relative path on public GitHub?

Reply:
Yes, Ember needs to be build with a different relative path for Github proper so that the build assets are able to be retrieved.